flatnav/util/SIMDIntrinsics.h

```diff
@@ -0,0 +1,631 @@
+#pragma once
```

Comment:
The name for this file is a little vague, perhaps SIMDDistanceSpecializations or similar.